packaging/opensuse/snapd.spec

```diff
@@ -168,83 +200,67 @@ export CFLAGS
 export CXXFLAGS
 export LDFLAGS
 
-# N.B.: Prior to openSUSE Tumbleweed in May 2018, the AppArmor userspace in SUSE
-# did not support what we needed to be able to turn on basic integration.
+# Generate autotools build system files.
+pushd %{indigo_srcdir}/cmd
+autoreconf -i -f
 %configure \
     %{!?with_apparmor:--disable-apparmor} \
```

Comment:
Wondering whether it'd be helpful to have an explicit `--enable-apparmor` for TW.

Reply:
Yeah, I can do that.

Reply:
Actually, I kept this at --disable-apparmor because we may have headers but the system still doesn't "do" apparmor the way we want.